src/Core/Constants.cs

```diff
@@ -47,6 +47,9 @@ public static class Constants
         public const int TotpDefaultTimer = 30;
         public const int PasswordlessNotificationTimeoutInMinutes = 15;
         public const int KdfIterations = 600000;
```

Comment:
Should this be renamed like was done on clients repo?

Reply:
I think it would be better to have the same naming on the constants config for `PBKDF2` and `Argon2` between here and clients repo. So it's easier to find and maintain cross-repo.